recipes/libmikmod/all/conanfile.py

```diff
@@ -0,0 +1,123 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibmikmodConan(ConanFile):
+    name = "libmikmod"
+    description = "Module player and library supporting many formats, including mod, s3m, it, and xm."
+    topics = ("libmikmod", "audio")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://mikmod.sourceforge.net"
+    license = "LGPL-2.1-or-later"
+    exports_sources = ["patches/*"]
+    generators = "cmake"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_dsound": [True, False],
+        "with_mmsound": [True, False],
+        "with_alsa": [True, False],
+        "with_oss": [True, False],
+        "with_pulse": [True, False],
+        "with_coreaudio": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_dsound": True,
+        "with_mmsound": True,
+        "with_alsa": True,
+        "with_oss": True,
+        "with_pulse": True,
+        "with_coreaudio": True
+    }
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        else:
+            del self.options.with_dsound
+            del self.options.with_mmsound
+        if self.settings.os != "Linux":
+            del self.options.with_alsa
+        # Non-Apple Unices
+        if self.settings.os not in ["Linux", "FreeBSD"]:
+            del self.options.with_oss
+            del self.options.with_pulse
+        # Apple
+        if tools.is_apple_os(self.settings.os):
+            del self.options.with_coreaudio
+
+    def configure(self):
+        del self.settings.compiler.libcxx
```

Comment:
Is theres a missing `Del fpic`

Reply:
I don't think so.
I see one in config_options and one in configure

Reply:
yes, I commited it after @prince-chrismc 's message